src/main/sensors/gyro.h

```diff
@@ -58,6 +58,11 @@ typedef enum {
     GYRO_OVERFLOW_CHECK_ALL_AXES
 } gyroOverflowCheck_e;
 
+typedef enum {
+    DYN_FILTER_BEFORE_STATIC_FILTERS = 0,
+    DYN_FILTER_AFTER_STATIC_FILTERS
+} dyn_filter_location_e;
```

Comment:
This should not be a `typedef`, since we are not using it as a type anywhere. And in that case the name can be dropped as well. Or if we want to keep it as a type, the name should be `dynFilterLocation_e` (i.e. camelCase with `_e` for enum).

Reply:
I don't get it, I'm sure I tried it all simple without typedef or name, and it wouldn't work.  Now it compiles perfectly!  I'm sorry for causing problems with this.  All good now.